src/coprocessor/metrics.rs

```diff
@@ -56,6 +56,17 @@ lazy_static! {
         &["req", "metric"]
     )
     .unwrap();
+    pub static ref COPR_DAG_REQ_COUNT: IntCounterVec = register_int_counter_vec!(
+        "tikv_coprocessor_dag_request_count",
+        "Total number of DAG requests",
+        &["vec_type"]
+    )
+    .unwrap();
+    pub static ref COPR_RESP_SIZE: IntCounter = register_int_counter!(
```

Comment:
Is simply a counter enough? Do we need a histogram?

Reply:
We mostly use it to observe whether or not Coprocessor is the cause of OOM, so a Counter that includes byte sizes of all requests is enough.

Reply:
Got it. Then, for OOM detection, is it better to add metrics of bytes already sent by gRPC (after sink sending finished)?

Reply:
Yes. This metrics will be added later by someone else.

Reply:
Currently we are able to recognize Coprocessor OOM by looking at metrics in this PR + Network outbound metrics.